bayesian/api_v1.py

```diff
@@ -340,22 +340,27 @@ def get(external_request_id):
         started_at = None
         finished_at = None
         manifest_response = []
-        recommendation = {}
+        recommendation = None
+        stack_data = None
 
-        if stack_result != None and 'task_result' in stack_result:
-            if stack_result["task_result"] != None:
-                started_at = stack_result["task_result"]["_audit"]["started_at"]
-                finished_at = stack_result["task_result"]["_audit"]["ended_at"]
+        if stack_result is not None and 'task_result' in stack_result:
```

Comment:
I think this whole block could be simplified so it's easier to read, for example:
```python
if stack_result and stack_result.get('task_result', None):
    task_result = stack_result["task_result"]

    stack_data = task_result.get('user_stack_info', None)
    started_at = task_result.get('_audit', {}).get('started_at', None)
    finished_at = task_result.get('_audit', {}).get('ended_at', None)
```

And it would also handle cases where there is `_audit` dictionary, but `started_at` or `ended_at` is missing.

But I can see that `if` statements with similar structure are used on multiple places in this module (not part of this PR), so let's keep it as it is, to be consistent.